keyboards/hineybush/h660s/keymaps/via/keymap.c

```diff
@@ -0,0 +1,80 @@
+/* Copyright 2020 Josh Hinnebusch
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE,
+    _FN,
+    _FN2
+};
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    QMKBEST = SAFE_RANGE,
+    QMKURL
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /*
+    k000, k001, k002, k003, k004, k005, k006, k007, k008, k009, k010, k011, k012, k113, k013,     k014, \
+    k100,   k101, k102, k103, k104, k105, k106, k107, k108, k109, k110, k111, k112,  k213,        k114, \
+    k200, k201, k202, k203, k204, k205, k206, k207, k208, k209, k210, k211, k212, k313,                 \
+    k300, k301, k302, k303, k304, k305, k306, k307, k308, k309, k310, k311, k312,           k314,       \
+    k400, k401, k402,       k407,                    k409, k410, k411,                k412, k413, k414  \
+    */
+
+    [_BASE] = LAYOUT_all(
```

Comment:
I believe you need a 4rd layer here, as it defaults to 4, and I don't see it changed to 3